doit/cmdparse.py

```diff
@@ -333,7 +333,7 @@ def parse_only(self, in_args, params=None):
         return params, args
 
 
-    def parse(self, in_args):
+    def parse(self, in_args, cfg_defaults={}):
```

Comment:
This name is a bit misleading, as it does not refer to the notion of "config file" and "defaults" would intuitively refer to a default value in the task definition, not in the config file.

I would suggest `cfg_from_file`, `cfg_from_doit_cfg_file`, `file_provided_cfg`, `cfgfile_args`, or similar.

Reply:
Thanks, I think you are right. I'll tweak that.